lib/conctracts.spec.ts

```diff
@@ -0,0 +1,30 @@
+import type { ContractDefinition } from '@balena/jellyfish-types/build/core';
```

Comment:
There's a typo in the name of this file. Not the import, this file

Reply:
oops...